microhapulator/workflows/analysis.smk

```diff
@@ -278,15 +274,14 @@ rule repetitive_mapping:
 
 rule read_mapping_qc:
     input:
-        marker=rules.map_sort_and_index.output.counts,
+        marker=rules.map_sort_and_index.output.stats,
         full_refr=rules.count_fullrefr_mapped_reads.output.counts,
         repetitive=rules.repetitive_mapping.output.counts,
     output:
         counts="analysis/{sample}/{sample}-read-mapping-qc.csv",
-        plot="analysis/{sample}/{sample}-donut.png",
     shell:
         """
-        mhpl8r mappingqc  --marker {input.marker} --refr {input.full_refr} --rep {input.repetitive}  --csv {output.counts}  --figure {output.plot} --title {wildcards.sample}
+        mhpl8r mappingqc {input.marker} {input.full_refr} {input.repetitive} --csv {output.counts}
```

Comment:
I made some changes to the mappingqc API and CLI. Required arguments are now declared as positional rather than keyword arguments, and the .csv and .png output are only generated when explicitly requested. The `mhpl8r mappingqc` command is still capable of generating the donut plots, but I've disabled it in the `mhpl8r pipe` workflow since these have been replaced with the stacked bar chart.